maven-repository-metadata/src/site/apt/index.apt

```diff
@@ -27,38 +27,8 @@ Maven Repository Metadata Model
 
  This is strictly the model for Maven Repository Metadata, so really just plain objects.
 
- The metadata file name is:
-
- * <<<maven-metadata.xml>>> in a remote repository,
-
- * <<<maven-metadata-\<repo-id>.xml>>> in a local repository, for metadata from a repository with <<<repo-id>>> identifier.
-
- []
-
- Depending on what the directory represents ("groupId", "groupId/artifactId" or "groupId/artifactId/version"),
- the Maven Repository Metadata file contains 3 different sets of metadata:
-
- [[1]] in a "groupId" directory: a "groupId" directory may contain Maven plugins artifacts, which are described in metadata's <<<plugins>>> element,
-
- [[2]] in a "groupId/artifactId" directory: metadata describes <<<groupId>>>, <<<artifactId>>> and <<<versioning>>> element that
-       gives data about available versions (<<<latest>>>, <<<release>>>, <<<versions>>> list and <<<lastUpdated>>>),
-
- [[3]] in a "groupId/artifactId/version" snapshot artifact directory: metadata describes <<<groupId>>>, <<<artifactId>>>, <<<version>>> (base version, i.e. ending in <<<-SNAPSHOT>>>) and
-       <<<versioning>>> element that gives data about snaphot (<<<snapshot>>>, <<<lastUpdated>>> and <<<snapshotVersions>>> list). Notice that a
-       release artifact directory is not expected to provide metadata.
-
- []
-
```

Comment:
If we document those things here, scattered (as filename is layout matter, caching names are LRM internal matters), this will just become yet another spot to update somewhere in the future, that I bet we will miss, directly leading just to even more "documentation deterioration"....

I mean, my original intent was to _remove_ IDE/compiler warning about use of (wrongly) deprecated classe., I will restore this page to it's original state, remove the deprecation message and add a link.

Reply:
Pushed a change that removes (wrong) sentence about deprecation (along with comments), and added link.

Reply:
sorry, I finally overlooked the last update: I think we have the best of every world now, thank you